src/format_meta.rs

```diff
@@ -49,7 +49,7 @@ impl<'a> Meta<'a> {
             let scaled_min = minmaxscale(range.min as usize, 0, buffer.len());
             let scaled_max = minmaxscale(range.max as usize, 0, buffer.len());
             let scaled_size = minmaxscale(size as usize, 0, buffer.len());
-            let mut chars = SCALE.clone();
```

Comment:
nice

Reply:
Explicitly cloning a type that implements `Copy` is actually useless to write, as the compiler does that already as a result of the assignment.